lib/js/styles/TokenTypes.ts

```diff
@@ -22,3 +24,9 @@ export interface TypographyToken {
 	attributes: Array<string>;
 	attributesRaw: Array<AttributeRaw>;
 }
+
+export interface ItemsListsItem {
+	title: string;
+	class: string;
+	list: Dict<object>;
```

Comment:
Why we can't specify type of this `object`?

Reply:
I will think about this i other PR when I will fixing everything + improved views for UX.